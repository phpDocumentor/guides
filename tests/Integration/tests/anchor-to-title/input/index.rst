.. _rst-overview:

Overview
**********

RST Overview content


.. _sphinx-overview:

Overview
*********

Sphinx Overview content
