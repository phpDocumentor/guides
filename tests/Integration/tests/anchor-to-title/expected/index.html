<!DOCTYPE html>
<html lang="en">
<head>


</head>

<body>
<div class="section" id="rst-overview">
    <h1>Overview</h1>
    <p>RST Overview content</p>
</div>
<div class="section" id="sphinx-overview">
    <h1>Overview</h1>
    <p>Sphinx Overview content</p>
</div>

</body>
</html>
